code/__HELPERS/type2type.dm

```diff
@@ -235,6 +235,19 @@ proc/tg_list2text(list/list, glue = ",")
 		if(SOUTHWEST)	return 225
 		else			return null
 
+//returns the east-zero counter-clockwise angle in degrees, given a direction
+
+/proc/dir2angle_t(const/D)
+	switch(D)
+		if(EAST)		return 0
```

Comment:
I like that you're aligning this code, but you should align it with spaces instead of tabs.